src/components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { isEmail } from 'validator';
+import { handleForgotPasswordForm, submitForgotPassword } from '../../redux/actions';
+import Input from '../common/Input/Input';
+import Button from '../common/Button/Button';
+
+export class ForgotPassword extends Component {
+  constructor() {
+    super();
+    this.state = {
+      validEmail: true,
+      emailError: 'Email is not valid',
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onSubmitButton = this.onSubmitButton.bind(this);
+    this.renderErrors = this.renderErrors.bind(this);
+  }
+
+  onSubmitButton(e) {
+    const { onSubmit, email } = this.props;
+    this.setState({ validEmail: true });
+    if (isEmail(email)) {
+      onSubmit({ email });
+    } else {
+      this.setState({ validEmail: false });
+    }
+    e.preventDefault();
+  }
+
+  handleInputChange(e) {
+    this.setState({ validEmail: true });
+    const { onInputChange } = this.props;
+    onInputChange({ field: e.target.name, value: e.target.value });
+  }
+
+  renderErrors() {
```

Comment:
Why are we not making use of ES6 ?

Reply:
@kimotho-njoki The Esling configuration that we are using does not allow arrow functions within a react component.
![image](https://user-images.githubusercontent.com/17350127/55684686-68acff80-594e-11e9-9847-73fb0bfac2a5.png)